splot/_viz_esda_mpl.py

```diff
@@ -13,14 +13,15 @@
 TODO
 geopandas plotting, change round shapes in legends to boxes
 change function input naming to be in line with bokeh functionality
+check if geopandas can read **kwargs
+check if attribute in gdf.plot works without attribute str
 """
 
 __author__ = ("Stefanie Lumnitz <stefanie.lumitz@gmail.com>")
 
 
-def moran_scatterplot(moran_loc, p=None, figsize=(7,7), xlabel='Attribute',
-                      ylabel='Spatial Lag', title='Moran Scatterplot',
-                      ax=None, alpha=0.6):
+def moran_loc_scatterplot(moran_loc, p=None,
```

Comment:
Not sure about the renaming of this function (I could be forgetting - or I missed - the discussion). `moran_scatterplot` seems cleaner (and matches the docstring).

Reply:
I renamed it because I started creating visualizations for other `esda` spatial autocorrelation objects like "Global" `Moran`, discussed before in https://github.com/pysal/splot/issues/10.

In order to distinguish visualizations for `Moran`, `Moran_Local`, `Moran_BV`, ... I renamed this one Ito `moran_loc_scatterplot`.

Here is a first draft of how the "Global" `Moran` visualization could look like:
![image](https://user-images.githubusercontent.com/35147471/41831089-e24a70ca-77f9-11e8-90e9-873fe4b7b9be.png)

Reply:
Thanks for clarifying. Looks good to me now.